Realm/ObjectStore/list.hpp

```diff
@@ -106,6 +106,8 @@ class List {
         size_t valid_count;
     };
 
+    LinkViewRef get_linkview() const { return m_link_view; }
```

Comment:
Exposing `get_anythreadconfined` seems a lot more gross to me, especially considering there's a `linkview` member function on `AnyThreadConfined`.